src/sentry/models/eventattachment.py

```diff
@@ -32,7 +32,7 @@ class EventAttachment(Model):
     class Meta:
         app_label = "sentry"
         db_table = "sentry_eventattachment"
-        index_together = (("project_id", "date_added"),)
+        index_together = (("project_id", "date_added", "file"),)
```

Comment:
This doe snot reflect what is happening here. This should just suggest that we're adding another index.

```
index_together = (
    ("project_id", "date_added",),
    ("project_id", "date_added", "file_id",),
)
```

is correct based on what we're doing. The migration is simply adding a new one. I think this then goes back to @wedamija's comment then about the state operation not matching.

Reply:
I actually suggested that he handle the migration in this way, just in terms of not having to be overly complex when changing the state. But if you prefer we could be more explicit and remove the old index in the followup pr.

Reply:
I'd prefer we do it correctly in two steps so the state reflects reality. It won't really hurt having the extra index, even if for example we never drop the other.

Reply:
@mattrobenolt I added the additional index. I'll create a follow up PR which removes the first index without the file.